app/src/common/translations/fr.json

```diff
@@ -0,0 +1,127 @@
+{
+  "common": {
+    "edit": "Modifier",
+    "save": "Enregistrer",
+    "cancel": "Annuler",
+    "close": "Fermer",
+    "clear": "Vider",
+    "download": "Télécharger",
+    "loading": "Chargement...",
+    "settings": "Paramètres",
+    "help": "Aide",
+    "documentation": "Documentation",
+    "headline": "Entête"
+  },
+  "theme": {
+    "headline": "Thème",
+    "default": "Défaut",
+    "dark": "Sombre"
+  },
+  "language": {
+    "headline": "Langage",
+    "en": "Anglais",
+    "de": "Allemand",
+    "fr": "Français"
+  },
+  "data": {
+    "purge": "Supprimer les données",
+    "backup": "Faire une sauvegarde",
+    "restore": "Restaurer une sauvegarde"
+  },
+  "welcome": {
+    "message1": "Bienvenue à <0>{{projectName}}</0> — Votre tableau de bord digital",
+    "message2": "Personnalisez le contenu en ajoutant de nouveaux composants depuis le menu ou utilisez les modèles suivants de composants.",
+    "message3": "Lisez la <0>documentation</0> pour plus d'informations.",
+    "createExampleBoard": "Utiliser un modèle"
+  },
+  "widget": {
+    "common": {
+      "drawerHeadline": "Composants",
+      "add": "Ajouter {{widget}} composant",
+      "remove": "Retirer le composant {{widget}}",
+      "configuration": "Configuration: {{widget}}",
+      "unconfigured": "Configuration requise."
+    },
+    "category": {
+      "general": "General",
+      "media": "Media",
+      "knowledge": "Knowledge",
+      "tools": "Tools",
+      "tracking": "Tracking",
+      "monitoring": "Monitoring"
+    },
+
+    "counter": {
+      "name": "Compteur",
+      "headline": "{{headline}}",
+      "increment": "Plus",
+      "decrement": "Moins"
+    },
+    "cryptocurrencies": {
+      "name": "Cryptomonnaie",
+      "headline": "",
+      "configuration": {
+        "fiat": "Fiat monnaie",
+        "crypto": "Crypto monnaie"
+      },
+      "crypto": {
```

Comment:
As cryptocurrency names don't change across languages, I would probably use the same fallback pattern as above (so by not duplicating the labels the "en" will be used as fallback). But I wouldn't consider it a "must fix".